setup.py

```diff
@@ -15,12 +18,13 @@
     author_email = "davidbarnett2@gmail.com",
     description = "The familiar example program in Python",
     license = "BSD",
+    long_description = long_description,
     keywords= "example documentation tutorial",
-    url = "http://github.com/dbarnett/python-helloworld",
+    url = "http://github.com/dbarnettpython-helloworld",
     entry_points = {
         "console_scripts": [        # command-line executables to expose
             "helloworld_in_python = helloworld.main:main",
         ],
-        "gui_scripts": []       # GUI executables (creates pyw on Windows)
+        "gui_scripts": []       # GUI exetables (creates pyw on Windows)
```

Comment:
This is misspelled. Was there a reason for changing it?

Reply:
No that's a typo